spec/metrics-spec.js

```diff
@@ -121,6 +121,26 @@ describe('Metrics', async () => {
       let url = Reporter.request.mostRecentCall.args[0]
       expect(url).toContain('aiid=stable')
     })
+
+    it('reports an arbitrary release channel', async () => {
+      spyOn(atom, 'getVersion').andReturn('1.0.2-sushi1')
+
+      await atom.packages.activatePackage('metrics')
+      await conditionPromise(() => Reporter.request.callCount > 0)
+
+      let url = Reporter.request.mostRecentCall.args[0]
+      expect(url).toContain('aiid=sushi')
+    })
```

Comment:
@daviwil: Since we know for sure that we want to support a nightly release channel, how would feel about adding a test specifically for the nightly release channel?

Reply:
Sure thing!